pages/games/[gameId].tsx

```diff
@@ -147,156 +152,218 @@ function GamePage(): JSX.Element {
+                fontWeight="normal"
+                lineHeight="40px"
+                _hover={{
+                  color: 'accent',
+                }}
               >
-                {shortenAddress(id)}
-              </Link>
-              <Heading fontSize="40px" fontWeight="normal" lineHeight="40px">
                 {name}
               </Heading>
-              <Text>{description}</Text>
+              <Text fontSize="xl" fontWeight={200} mb={2}>
+                {description}
+              </Text>
+              <HStack spacing={4}>
+                <Link
+                  fontSize="sm"
+                  href={`${EXPLORER_URLS[chainId]}/address/${id}`}
+                  isExternal
+                  fontWeight={300}
+                  mb={3}
+                  textDecoration={'underline'}
+                >
+                  {shortenAddress(id)}
+                </Link>
+                {isMaster && (
+                  <Button onClick={updateGameMetadata.onOpen} size="sm">
+                    <Flex align="center" gap={2}>
+                      <Image
+                        alt="edit"
+                        height="14px"
+                        src="/icons/edit.svg"
+                        width="14px"
+                      />
+                      Edit
+                    </Flex>
+                  </Button>
+                )}
+              </HStack>
             </VStack>
-            {isConnectedAndMounted && !character && (
-              <Button onClick={joinGameModal.onOpen}>Join this Game</Button>
-            )}
-            {isConnectedAndMounted && character && character.removed && (
-              <Button onClick={restoreCharacterModal.onOpen}>
-                Restore Character
-              </Button>
-            )}
-
-            {isMaster && (
-              <Button onClick={updateGameMetadata.onOpen} size="sm">
-                <Flex align="center" gap={2}>
-                  <Image
-                    alt="edit"
-                    height="14px"
-                    src="/icons/edit.svg"
-                    width="14px"
-                  />
-                  Edit
-                </Flex>
-              </Button>
-            )}
           </HStack>
-          <Image
-            alt="game emblem"
-            background="gray.400"
-            h="140px"
-            objectFit="cover"
-            src={image}
-          />
+          <VStack
+            align="start"
+            spacing={0}
+            h="100%"
+            bg="cardBG"
+            flexShrink={0}
+            p={8}
+          >
+            <GameTotals
+              experience={experience}
+              characters={characters}
+              items={items}
+            />
+          </VStack>
         </HStack>
 
-        <VStack align="start" pb={10} px={14} w="full">
-          <Text fontSize="lg" fontWeight="bold">
-            GameMasters
+        <VStack align="start" spacing={4} p={8} bg="cardBG">
```

Comment:
Ben also has the owner of the game listed at the top here. I thought it would be nice to have the RaidGuild DAO, for instance, listed separately than all the other GameMaster bots, but did you have a reason for not including it?

Reply:
Added in 5fff41d32f2580a90e7ca648147de5f3bd57535c